README.md

```diff
@@ -52,18 +53,17 @@ __Code Status__ [^1]
-
-[![TO Go Client Integration Tests](https://github.com/apache/trafficcontrol/workflows/TO%20Go%20Client%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/traffic-ops.yml?query=branch%3Amaster) [![Codecov](https://codecov.io/gh/apache/trafficcontrol/branch/master/graph/badge.svg?flag=traffic_ops_integration)](https://app.codecov.io/github/apache/trafficcontrol/tree/master/traffic_ops)
-
-[![Traffic Router Tests](https://github.com/apache/trafficcontrol/workflows/Traffic%20Router%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tr.tests.yaml?query=branch%3Amaster) [![Codecov](https://codecov.io/gh/apache/trafficcontrol/branch/master/graph/badge.svg?flag=traffic_router_unit)](https://app.codecov.io/github/apache/trafficcontrol/tree/master/traffic_router)
-
-[![TP Integration Tests](https://github.com/apache/trafficcontrol/workflows/TP%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tp.integration.tests.yml?query=branch%3Amaster) 
-[![TM Integration Tests](https://github.com/apache/trafficcontrol/workflows/TM%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tm.integration.tests.yml?query=branch%3Amaster)
-[![TR Ultimate Test Harness](https://github.com/apache/trafficcontrol/workflows/TR%20Ultimate%20Test%20Harness/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/tr-ultimate-test-harness.yml?query=branch%3Amaster)
-[![T3C Integration Tests](https://github.com/apache/trafficcontrol/workflows/T3C%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/cache-config-tests.yml?query=branch%3Amaster)
-[![TC Health Client Integration Tests](https://github.com/apache/trafficcontrol/workflows/TC%20Health%20Client%20Integration%20Tests/badge.svg)](https://github.com/apache/trafficcontrol/actions/workflows/health-client-tests.yml?query=branch%3Amaster)
-
+| Unit Tests                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Integration Tests                                                                                                                                                                                                                                                                                                                                                                                                                            | 
```

Comment:
Do you think this table should have a "Component" column? So I can easily understand why two badges are on the same line, and what the dashes represent.